index.html

```diff
@@ -129,8 +129,14 @@ <h4>Card with shadow</h4>
         <p>Description</p>
         <button>Some Button</button>
     </div>
-
     <hr>
+    <h2>Image Component</h2>
+
+    <h4>Responsive Image</h4>
+    <img src="https://placehold.co/200x200" class="img-responsive"/>
+
+    <h4>Rounded Image</h4>
+    <img src="https://placehold.co/200x200" class="img-rounded"/>
```

Comment:
You can make use of Alt tag to make your image component Accessible for everyone.